### PR TITLE
implemented the sensor page

### PR DESCRIPTION
**Pull request by cmanamela**
Please review my code

### CODE REVIEW

File: waterlog/src/Stylesheets/_sensor.scss [hunk 1]
```diff
@@ -0,0 +1,43 @@
+#sensor-diagram {
```
Comment (FredJEhlers) [inline waterlog/src/Stylesheets/_sensor.scss:1]: inconsistent when compared to `waterlog/src/App.scss`

Comment (FredJEhlers) [reply waterlog/src/Stylesheets/_sensor.scss:1]: Also add empty line EOF

**Review by FredJEhlers — CHANGES_REQUESTED** [review]

File: waterlog/src/Stylesheets/_sensor.scss (hunk 1 above)
Comment (cmanamela) [reply waterlog/src/Stylesheets/_sensor.scss:1]: Fixed

Comment (cmanamela) [reply waterlog/src/Stylesheets/_sensor.scss:1]: tabs are the same in both the files, so i only managed to sort out the first one

**Review by FredJEhlers — APPROVED** [review]

**Pull request merged**